snippets/ztp_example-sno.yaml

```diff
@@ -0,0 +1,127 @@
+kind: SiteConfig
+metadata:
+  name: "example-sno"
+  namespace: "example-sno"
+spec:
+  baseDomain: "example.com"
+  cpuPartitioningMode: AllNodes
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  - clusterName: "example-sno"
+    networkType: "OVNKubernetes"
+    installConfigOverrides: |
+      {
+        "capabilities": {
+          "baselineCapabilitySet": "None",
+          "additionalEnabledCapabilities": [
+            "marketplace",
+            "NodeTuning"
+          ]
+        }
+      }
+    clusterLabels:
+      common: true
+      group-du-sno: ""
+      sites : "example-sno"
+    clusterNetwork:
+      - cidr: 1001:1::/48
+        hostPrefix: 64
+    machineNetwork:
+      - cidr: 1111:2222:3333:4444::/64
+    serviceNetwork:
+      - 1001:2::/112
+    additionalNTPSources:
+      - 1111:2222:3333:4444::2
+    # crTemplates:
+    #   KlusterletAddonConfig: "KlusterletAddonConfigOverride.yaml"
+    nodes:
+      - hostName: "example-node1.example.com"
+        role: "master"
+        # biosConfigRef:
+        #   filePath: "example-hw.profile"
+        bmcAddress: "idrac-virtualmedia+https://[1111:2222:3333:4444::bbbb:1]/redfish/v1/Systems/System.Embedded.1"
+        bmcCredentialsName:
+          name: "example-node1-bmh-secret"
+        bootMACAddress: "AA:BB:CC:DD:EE:11"
+        bootMode: "UEFI"
+        rootDeviceHints:
+          wwn: "0x11111000000asd123"
+        # diskPartition:
+        #   - device: /dev/disk/by-id/wwn-0x11111000000asd123 # match rootDeviceHints
+        #     partitions:
+        #       - mount_point: /var/imageregistry
+        #         size: 102500
+        #         start: 344844
+        ignitionConfigOverride: |
+          {
+            "ignition": {
+              "version": "3.2.0"
+            },
+            "storage": {
+              "disks": [
+                {
+                  "device": "/dev/disk/by-id/wwn-0x11111000000asd123",
+                  "wipeTable": false,
+                  "partitions": [
+                    {
+                      "sizeMiB": 16,
+                      "label": "httpevent1",
+                      "startMiB": 350000
+                    },
+                    {
+                      "sizeMiB": 16,
+                      "label": "httpevent2",
+                      "startMiB": 350016
+                    }
+                  ]
+                }
+              ],
+              "filesystem": [
+                {
+                  "device": "/dev/disk/by-partlabel/httpevent1",
+                  "format": "xfs",
+                  "wipeFilesystem": true
+                },
+                {
+                  "device": "/dev/disk/by-partlabel/httpevent2",
+                  "format": "xfs",
+                  "wipeFilesystem": true
+                }
+              ]
+            }
+          }
+        cpuset: "0-1,20-21"
```

Comment:
cpuset attribute need to be removed as we set "cpuPartioningMode: All nodes" in this example.

Reply:
QE review completed.
/Peri M